oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -98,7 +98,8 @@ storage:
     password: ${SW_ES_PASSWORD:""}
     secretsManagementFile: ${SW_ES_SECRETS_MANAGEMENT_FILE:""} # Secrets management file in the properties format includes the username, password, which are managed by 3rd party tool.
     dayStep: ${SW_STORAGE_DAY_STEP:1} # Represent the number of days in the one minute/hour/day index.
-    indexShardsNumber: ${SW_STORAGE_ES_INDEX_SHARDS_NUMBER:2}
+    indexShardsNumber: ${SW_STORAGE_ES_INDEX_SHARDS_NUMBER:1}
```

Comment:
This should be kept in 2, endpoint metrics could be a big size too.

Reply:
Metrics may the data reach 30G per day, if it is below this, it can be adjusted to 1 for mostly users

Reply:
I am not following your meaning. Could you be more clear?

Reply:
the following image is a index in our preonline  environmen(the number of endpoint is about 203)t:
![image](https://user-images.githubusercontent.com/31562192/82138128-ac13b600-9850-11ea-8810-19846749bf8a.png)

in 0516, there are have 124060 docs, and the size of it is about 13.6M

From the data above, it seems that 30G can already store a lot of metrics data
Unless it is a very large system, I don't think it will exceed 30G

In es 6 ,the default shards number is 5
in es 7,the default shards number is 1 ,The reason is that the shards setting is unreasonable

So I think  It's enough to set one shards for skywalking metrics data

Reply:
OK, make sense. Please follow other comments to continue.

Reply:
ok

Reply:
Is there a checkstyle file for skywalking, I want to set it in idea

Reply:
Yes, of course. It is inside the root folder.